File: SAV_Backend/SAV_Backend/Models/ResponsableSAV.cs
using System.IO.Pipelines;

namespace SAV_Backend.Models
{
    public class ResponsableSAV
    {
        public int Id { get; set; }
        public string Nom { get; set; }
        public string Prenom { get; set; }
        public string Email { get; set; }
        public string Telephone { get; set; }
        public string? ApplicationUserId { get; set; } 
        public ApplicationUser? ApplicationUser { get; set; }
        public ICollection<Intervention>? Interventions { get; set; }
        

    }



}

File: SAV_Frontend/src/Models/ResponsableSAV.ts
import { Intervention } from "./Intervention";

export interface ResponsableSAV {
    id: number;
    nom: string;
    prenom: string;
    email: string;
    telephone: string;
    applicationUserId?: string; // Propriété facultative
    applicationUser?: any; // Remplacez 'any' par l'interface appropriée si vous avez un modèle d'utilisateur
    interventions?: Intervention[]; // Remplacez 'Intervention' par l'interface appropriée si elle existe
  }
  